pos_discount_absolute/static/src/js/pos_discount_absolute.js

```diff
@@ -16,8 +16,10 @@ odoo.define('pos_discount_absolute', function (require) {
         show: function (options) {
             var self = this;
             this._super(options);
+            // popup_abs_discount prevents errors on a page loading and exclude discount stuff rendering in other popups
             this.popup_abs_discount = false;
-            if (this.pos.config.discount_abs_enabled) {
+            this.pos.discount_abs_type = false;
+            if (this.pos.config.discount_abs_enabled && options.title === "Discount Percentage") {
```

Comment:
You shall not trust to "title" value, because it may have translated value. Try to find another solution